jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -0,0 +1,294 @@
+    sort_order: SortOrder = typer.Option(SortOrder.UNSORTED, help="Specify sort order"),
+    sort_field: Optional[str] = typer.Option(None, help="The field by which results should be sorted"),
+):
+    """
+    Show available job scripts
+    """
+    jg_ctx: JobbergateContext = ctx.obj
+
+    # Make static type checkers happy
+    assert jg_ctx is not None
+    assert jg_ctx.client is not None
+
+    params: Dict[str, Any] = dict(all=show_all)
+    if search is not None:
+        params["search"] = search
+    if sort_order is not SortOrder.UNSORTED:
+        params["sort_ascending"] = SortOrder is SortOrder.ASCENDING
+    if sort_field is not None:
+        params["sort_field"] = sort_field
+
+    envelope = cast(
+        ListResponseEnvelope,
+        make_request(
+            jg_ctx.client,
+            "/job-scripts",
+            "GET",
+            expected_status=200,
+            abort_message="Couldn't retrieve job scripts list from API",
+            support=True,
+            response_model=ListResponseEnvelope,
+            params=params,
+        ),
+    )
+    render_list_results(
+        jg_ctx,
+        envelope,
+        title="Job Scripts List",
+        style_mapper=style_mapper,
+        hidden_fields=HIDDEN_FIELDS,
+    )
+
+
+@app.command()
+@handle_abort
+def get_one(
+    ctx: typer.Context,
+    id: int = typer.Option(int, help="The specific id of the job script."),
+):
+    """
+    Get a single job script by id
+    """
+    jg_ctx: JobbergateContext = ctx.obj
+    result = fetch_job_script_data(jg_ctx, id)
+    render_single_result(
+        jg_ctx,
+        result,
+        hidden_fields=HIDDEN_FIELDS,
+        title="Job Script",
+    )
+
+
+@app.command()
+@handle_abort
+def create(
+    ctx: typer.Context,
+    name: str = typer.Option(
+        ...,
+        help="The name of the job script to create",
+    ),
+    application_id: Optional[int] = typer.Option(
+        None,
+        help="The id of the application from which to create the job script",
+    ),
+    application_identifier: Optional[str] = typer.Option(
+        None,
+        help="The identifier of the application from which to create the job script",
+    ),
+    sbatch_params: Optional[List[str]] = typer.Option(
+        None,
+        help="Optional parameter to submit raw sbatch parameters",
+    ),
+    param_file: Optional[pathlib.Path] = typer.Option(
+        None,
+        help=snick.dedent(
+            """
+            Supply a yaml file that contains the parameters for populating templates.
+            If this is not supplied, the question asking in the application is triggered.
+            """
+        ),
+    ),
+    fast: bool = typer.Option(
+        False,
+        help="Use default answers (when available) instead of asking the user.",
+    ),
+    submit: bool = typer.Option(
+        True,
+        help="Do not ask the user if they want to submit a job",
```

Comment:
```suggestion
        help="The name of the job script to create.",
    ),
    application_id: Optional[int] = typer.Option(
        None,
        help="The id of the application from which to create the job script.",
    ),
    application_identifier: Optional[str] = typer.Option(
        None,
        help="The identifier of the application from which to create the job script.",
    ),
    sbatch_params: Optional[List[str]] = typer.Option(
        None,
        help="Optional parameter to submit raw sbatch parameters.",
    ),
    param_file: Optional[pathlib.Path] = typer.Option(
        None,
        help=snick.dedent(
            """
            Supply a yaml file that contains the parameters for populating templates.
            If this is not supplied, the question asking in the application is triggered.
            """
        ),
    ),
    fast: bool = typer.Option(
        False,
        help="Use default answers (when available) instead of asking the user.",
    ),
    submit: bool = typer.Option(
        True,
        help="Do not ask the user if they want to submit a job.",
```

Reply:
Addressed in latest commit.